cfme/tests/containers/test_smart_management.py

```diff
@@ -74,29 +82,39 @@ def wait_for_tag(obj_inst):
                                forced_streams=['5.7'],
                                unblock=lambda test_item: test_item.obj != Container)])
 def test_smart_management_add_tag(provider, test_item):
-
+    logger.info("Setting smart mgmt tag to {obj_type}".format(obj_type=test_item.obj.__name__))
     # validate no tag set to project
     if test_item.obj is ContainersProvider:
         obj_inst = provider
     else:
         obj_inst = test_item.obj.get_random_instances(provider, count=1).pop()
 
+    logger.info("Selected object is \"{obj_name}\"".format(obj_name=obj_inst.name))
+
     regex = r"([\w\s|\-|\*]+:([\w\s|\-|\*])+)|(No.*assigned)"
     try:
         # Remove all previous configured tags for given object
+        logger.info('Starting cleaning old tags from '
+                    'object "{obj_name}"'.format(obj_name=obj_inst.name))
         obj_inst.remove_tags(obj_inst.get_tags())
+        logger.info("All smart management tags was removed successfully")
     except RuntimeError:
+        logger.info("Fail to remove tags, checking if no tag set")
+
         # Validate old tags formatting
-        assert re.match(regex, wait_for_tag(obj_inst).text_value), \
-            "Tag formatting is invalid! "
+        assert re.match(regex, wait_for_tag(obj_inst).text_value), (
+            "Tag formatting is invalid! ")
+        logger.info("No tag was set, continuing to main test")
```

Comment:
Not very useful IMHO.

Reply:
This needed for cases that exception occur during tag removal (and this happened to be a lot , may be FW issue) and i want to verify that no tag was exist on the selected object.

Reply:
@mkoura I don't understand where it strand? Do i have to fix more stuff or this PR is approved?